.github/workflows/weekly_update.yml

```
@@ -0,0 +1,39 @@
+name: Autoupdate
+
+on:
+  schedule:
+    - cron: "0 0 * * 1"  # every Monday at midnight
```

We could set
```suggestion
on:
  workflow_dispatch
```
to trigger the workflow manually for testing purposes.

---

We could also add this permanently to leave the possibility to trigger the workflow manually whenever a manual update (e.g., off the weekly period) is needed. 

This could allow us to drop the scripts weeklyUpdate.py and releasePR.py for the PR automation part - less code to maintain.

---

Done in [d3f28e3](https://github.com/BLAST-WarpX/warpx/pull/6054/commits/d3f28e3c708255d9c0ddc0770e291c68ef0ea648).